.travis.yml

```diff
@@ -35,12 +35,12 @@ before_script:
 - for key in $keys; do echo "$key=${!key}" >> .prod.env; echo "" >> .prod.env; done
 - echo "DATABASE_URL=mysql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}/${DATABASE_NAME}" >> .prod.env
 - echo "" >> .prod.env
-- git clone https://$GITHUB_ACCESS_TOKEN@github.com/masschallenge/directory ../directory
+- git clone -b AC-5743-2 https://$GITHUB_ACCESS_TOKEN@github.com/masschallenge/directory ../directory
```

Comment:
This change should be removed before merging, but I need it to deploy the directory branch to `test4`